src/AppHeader.tsx

```diff
@@ -55,7 +57,18 @@ export const AppHeader: React.FC = () => {
             </span>
             <CartIcon className="appheader__carticon" />
           </button>
+          {showCartPopup && (
+            <div className="appheader__cartpopup">
+              <p>{t('cart-popup-info', {quantity: quantity.toString()})}</p>
+              <button className="epbtn" onClick={handleCartModal}>{t('view-cart')}</button>
+            </div>
+          )}
         </div>
+        {isLoggedIn && (
```

Comment:
Confirmed with @BonnieEP, we should be displaying this for guest users too. We can remove this check.